server/src/com/cloud/configuration/ConfigurationManagerImpl.java

```diff
@@ -3295,8 +3296,8 @@ public VlanVO doInTransaction(final TransactionStatus status) {
                     _resourceLimitMgr.incrementResourceCount(vlanOwner.getId(), ResourceType.public_ip, new Long(ips.size()));
                 } else if (domain != null) {
                     // This VLAN is domain-wide, so create a DomainVlanMapVO entry
-                    final DomainVlanMapVO domainVlanMapVO = new DomainVlanMapVO(domain.getId(), vlan.getId());
-                    _domainVlanMapDao.persist(domainVlanMapVO);
+                    //final DomainVlanMapVO domainVlanMapVO = new DomainVlanMapVO(domain.getId(), vlan.getId());
```

Comment:
Hi Wei @ustcweizhou,
I would like to ask you about persisting entries on `doman_vlan_map` table. These lines were due to a quick fix for fetching public IP for system vms range, as per search builder on `VlanDaoImpl` does not consider vlans which are mapped on `domain_vlan_map` only returning range for vlan id = 1. Can you check my last comment on this PR as well?

Reply:
I think it will break the something obviously.